src/styles/card.scss

```diff
@@ -0,0 +1,6 @@
+.card {
+  display: flex;
+  width: $cardWidth;
```

Comment:
The variable `$cardWidth` is used but not defined. Ensure it is defined in your SCSS files to avoid compilation errors.